boards/arm/nrf5340_audio_dk_nrf5340/nrf5340_audio_dk_cpunet_reset.c

```diff
@@ -0,0 +1,144 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/kernel.h>
+#include <zephyr/init.h>
+#include <zephyr/logging/log.h>
+
+#include <soc.h>
+#include <zephyr/drivers/gpio.h>
+
+#include <nrfx_gpiote.h>
+
+LOG_MODULE_REGISTER(nrf5340_audio_dk_nrf5340_cpuapp, CONFIG_LOG_DEFAULT_LEVEL);
+
+#if defined(CONFIG_BT_CTLR_DEBUG_PINS_CPUAPP)
+#include <../subsys/bluetooth/controller/ll_sw/nordic/hal/nrf5/debug.h>
+#else
+#define DEBUG_SETUP()
+#endif
+
+static int core_config(void)
+{
+	int ret;
+	nrf_gpiote_latency_t latency;
+
+	latency = nrfx_gpiote_latency_get();
+
+	if (latency != NRF_GPIOTE_LATENCY_LOWPOWER) {
+		LOG_DBG("Setting gpiote latency to low power");
+		nrfx_gpiote_latency_set(NRF_GPIOTE_LATENCY_LOWPOWER);
+	}
+
+	/* USB port detection
+	 * See nPM1100 datasheet for more information
+	 */
+	static const struct gpio_dt_spec pmic_iset =
+		GPIO_DT_SPEC_GET(DT_NODELABEL(pmic_iset_out), gpios);
+
+	if (!device_is_ready(pmic_iset.port)) {
+		LOG_ERR("GPIO is not ready!");
+		return -ENXIO;
```

Comment:
`-ENODEV`

Reply:
Why do you want -ENODEV here? Do you want to change the other -ENXIOs as well?

Reply:
Yes